packages/client/src/components/forms/CdfInput.re

```diff
@@ -487,10 +487,10 @@ module Main = {
       <div className=Styles.chartSection>
         {switch (state.dataType, state.cdfType) {
          | ("FLOAT_CDF", _) when E.A.length(state.floatCdf.xs) > 1 =>
-           <FC__CdfChart__Large cdf={state.floatCdf} />
+           <FC__CdfChart__Large cdf={state.floatCdf} width=Some(0) />
```

Comment:
Why is the width 0? That seems odd to me.

Reply:
It was done to save backward compatibility. Later we could refactor this a bit more.